### PR TITLE
Wait for page to load before checking for consignmentId in url

### PR DESCRIPTION
**Pull request by techncl**

### CODE REVIEW

File: src/test/scala/steps/Steps.scala [hunk 1]
```diff
@@ -199,9 +199,16 @@ class Steps extends ScalaDsl with EN with Matchers {
 
   Then("^the user should be on a page with (.*) and a consignmentId in the URL") {
     page: String =>
+      new WebDriverWait(webDriver, 30).until((driver: WebDriver) => {
+        val currentUrl: String = webDriver.getCurrentUrl
+        val elementInUrl = currentUrl.split("/").takeRight(2).head
+        val consignmentTypes = Set("judgment", "consignment")
+        !consignmentTypes.contains(elementInUrl) // if element is not "judgment" nor "consignment", then it's probably the consignmentId
```
Comment (TomJKing) [inline src/test/scala/steps/Steps.scala:206]: Why is this needed here?

**Review by TomJKing — COMMENTED** [review]
Looks good. Just one question.

File: src/test/scala/steps/Steps.scala (hunk 1 above)
Comment (techncl) [reply src/test/scala/steps/Steps.scala:206]: Do you mean the comment or the `!consignmentTypes.contains(elementInUrl)`? If it's the comment, it's because I wasn't sure it was clear to someone reading why it's checking for "judgment" or "consignment" in the url as opposed to the consignmentId; I can remove it if it's clear. If it's the code you're talking about, it's because the function needs to return a boolean.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks
